openapi.yaml

```
@@ -89,6 +89,16 @@ paths:
           description: latitude, longitude in degrees
           schema:
             type: string
+        - name: filterType
+          in: query
+          required: false
+          description: |
+            Optional. Default is all types.
+            
+            Only return results of the given types.
```

(not a maintainer)

Looking at the code, this is likely a filter which can only be applied once => can only return one type

There is an occurance of this below too.

```suggestion
            Only return results of the given type.
```